### DISCUSSION TITLE
Please add Briefer as a one-click deploy!

### DISCUSSION THREAD

**Discussion by finnicius** [coollabsio/coolify#4411] [Service Template Requests]
Hi Coolify Team,

I’d like to request the addition of Briefer as a one-click deploy service template in Coolify.

Briefer (https://briefer.cloud/) is an open-source, collaborative platform that can be considered a Jupyter on steroids, designed to redefine how teams work with notebooks. It combines SQL, Python, and dashboards into a seamless environment for data manipulation, visualization, and sharing. Briefer empowers users to transform notebooks into dashboards or fully interactive data apps, making it an exceptional tool for data scientists, engineers, and analytics teams.

### Key Features of Briefer:

- **Jupyter Reimagined**: All the functionality of Jupyter, enhanced with real-time collaboration, interactive dashboard creation, and AI-powered code assistance.
- **SQL and Python Integration**: Query data directly with SQL, process it as DataFrames in Python, and build insightful outputs.
- **Interactive Dashboards**: Create dashboards and apps with filters, dropdowns, and input fields.
- **Collaboration Tools**: Real-time multiplayer editing, comments, and granular permissions for team projects.
- **Scheduling and Notifications**: Automate reports and send notifications to email or Slack.
- **Pre-installed Libraries and AI Assistant**: Accelerate development with hundreds of libraries and AI-powered coding assistance.

### Why Coolify?

Briefer is built for developers and data professionals who value seamless deployment and usability. Adding Briefer to Coolify would allow users to quickly deploy this next-generation notebook platform in their preferred environments, enhancing accessibility for both teams and individual contributors.

You can find more details about Briefer on GitHub (https://github.com/briefercloud/briefer) or its official site (https://briefer.cloud/).

Thanks for considering this request!

**Comment by mredodos** [comment]
+1

**Comment by arthrod** [comment]
@finnicius hey man! you said you were able to make it work for a bit, how did you do it?
